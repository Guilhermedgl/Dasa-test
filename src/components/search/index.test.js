import React from 'react';
import { create } from 'react-test-renderer';
import Search from './index';

describe('<Search />', () => {
  const handleChange = jest.fn();

  it('should snapshot should match', () => {
    const component = create(<Search handleChange={handleChange} />);
    expect(component.toJSON()).toMatchSnapshot();
  });

  it('should not use class', () => {
    const component = create(<Search handleChange={handleChange} />);
    expect(component.toJSON().props.class).toBeFalsy();
  });
  
  it('should receive handleChange props', () => {
    const component = create(<Search handleChange={handleChange} />);
    expect(component.toTree().props.handleChange).toBeTruthy();
  });
});