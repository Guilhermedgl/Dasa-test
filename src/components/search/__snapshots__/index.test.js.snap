// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> should snapshot should match 1`] = `
<div
  className="container-input"
>
  <input
    className="input"
    onChange={[MockFunction]}
    placeholder="Pesquise por um Pokemon"
    type="text"
  />
</div>
`;
